lib/tests/test.js

```diff
@@ -349,48 +346,20 @@ class Test {
     });
   }
 
-  require(module) {
-    if (module.startsWith('Embark/contracts/')) {
-      const contractName = module.substr(17);
-      if (this.contracts[contractName]) {
-        return this.contracts[contractName];
-      }
```

Comment:
what's up with all the code removed here? was it simplified somewhat?

Reply:
If you look at `createContractObject` the 2 functions were quite similar and seemed to have the same purpose:
initialising the contract.

However, now the require function is happening before there is a connection and before the contract are compiled.
So the require returns only a reference of what will be the contract once deployed:
```
let newContract = {};
this.contracts[contractName] = newContract;
```

Then the `newContract` gets populated into `createContractObject`

Let me know if you think I miss something